src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BytesKmerizer.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * Iterator over successive Kmers from a byte[].
+ * Bytes are interpreted as ASCII characters.
+ * Silently skips over parts of the sequence that has characters other than A, C, G, or T.
+ *
+ * Created by tsharpe on 12/18/15.
+ */
+public class BytesKmerizer implements Iterator<Kmer>
+{
+    private final byte[] seq;
+    private final int kSize;
+    private int idx;
+    private Kmer kmer;
+
+    public BytesKmerizer( final byte[] someSeq, final int someKSize )
```

Comment:
My personal taste is that this.xyzzy = xyzzy is error prone and noisy, but I bow to local convention.  Next check-in will eliminate "some" in all 5 files.